syz-fuzzer/fuzzer.go

```diff
@@ -560,3 +562,16 @@ func parseOutputType(str string) OutputType {
 		return OutputNone
 	}
 }
+
+// Currently, only use GetWeightedPCsRes to check if filter enabled.
+// Weighted PC table will not be used.
+func (fuzzer *Fuzzer) getWeightedPCs() {
+	a := &rpctype.GetWeightedPCsArgs{}
+	r := &rpctype.GetWeightedPCsRes{EnableFilter: false}
+	if err := fuzzer.manager.Call("Manager.GetWeightedPCs", a, r); err != nil {
+		log.Fatalf("Manager.GetWeightedPCs call failed: %v", err)
+	}
+	if r.EnableFilter {
```

Comment:
I think it will be much simpler and less code if we return this flag in ConnectRes.